CHANGES

```diff
@@ -0,0 +1,19 @@
+Revision history for SPIRV-Tools
+
+v2016.0wip 2016-04-21
```

Comment:
I'd suggest to remove the time tag on this release notes; from my understanding, at any time, there is only one `wip` "version"---the `master` branch. It will change frequently, and eventually be in a state to be tagged as the next release. A release should have a time tag, but the ongoing `master` dev branch should not, I think. Having a time tag for it just gives promise to update the time tag every time we put something new in the `master` branch.